pymeasure/instruments/panasonic/panasonicVP7722A.py

```diff
@@ -451,6 +451,32 @@ def measure_range_auto(self):
         """,
     )
 
+    trigger_measurement = Instrument.measurement(
```

Comment:
A note: "trigger" is a verb here, which indicates an action, which suggests to use a method, not a property. This should probably be called `triggered_measurement`, a triggered measurement is what is being returned.